docs/community/software/index.mdx

```diff
@@ -11,5 +11,7 @@ Current community projects:
 
 - [Meshtasticator (Simulator)](/docs/community/software/community-meshtasticator) - Meshtasticator is a discrete-event and interactive simulator that mimics the radio section of the device software.
 - [Meshtastic Web API](/docs/community/software/meshtastic-web-api) - Meshtastic Web API provides a RESTful interface to interact with a Meshtastic node via a serial connection.
+- [Mesh-metrics](/docs/community/software/community-mesh-metrics) - Meshtastic repeater metrics scraper using meshtastic-cli to be ingested by node_exporter for use in Prometheus//VictoriaMetrics TDB with a Grafana dashboard and sample alerting rules.
```

Comment:
is there an extra '/' between Prometheus and VictoriaMetrics?  Is VictoriaMetrics one or two words?  What does TBD mean in this context?  Thanks!

Reply:
Removed the forward slash's completely.